selfdrive/car/ford/fordcan.py

```diff
@@ -128,17 +128,16 @@ def create_acc_msg(packer, CAN: CanBus, long_active: bool, gas: float, accel: fl
   Frequency is 50Hz.
   """
 
-  decel = accel < 0 and long_active
   values = {
     "AccBrkTot_A_Rq": accel,                          # Brake total accel request: [-20|11.9449] m/s^2
     "Cmbb_B_Enbl": 1 if long_active else 0,           # Enabled: 0=No, 1=Yes
-    "AccPrpl_A_Rq": gas,                              # Acceleration request: [-5|5.23] m/s^2
+    "AccPrpl_A_Rq": -5.0,                             # Acceleration request: [-5|5.23] m/s^2
     "AccPrpl_A_Pred": gas,                            # Acceleration request: [-5|5.23] m/s^2
     "AccResumEnbl_B_Rq": 1 if long_active else 0,
     "AccVeh_V_Trg": v_ego_kph,                        # Target speed: [0|255] km/h
     # TODO: we may be able to improve braking response by utilizing pre-charging better
-    "AccBrkPrchg_B_Rq": 1 if decel else 0,            # Pre-charge brake request: 0=No, 1=Yes
-    "AccBrkDecel_B_Rq": 1 if decel else 0,            # Deceleration request: 0=Inactive, 1=Active
+    "AccBrkPrchg_B_Rq": 0,                            # Pre-charge brake request: 0=No, 1=Yes
+    "AccBrkDecel_B_Rq": 0,                            # Deceleration request: 0=Inactive, 1=Active
```

Comment:
It can brake if you don't set these?

Reply:
EVs will have absolutely 0 brake. There is no engine braking OR regen braking without these set. While Ford EVs are not currently supported in Upstream due to injection testing, if someone unflags their car out of dashcam mode this will not respond to any slow down attempts.